demo/index.js

```diff
@@ -10,6 +10,7 @@ import * as svg2pdf from '../node_modules/svg2pdf.js/dist/svg2pdf.min';
     var openSheetMusicDisplay;
     var sampleFolder = process.env.STATIC_FILES_SUBFOLDER ? process.env.STATIC_FILES_SUBFOLDER + "/" : "",
         samples = {
+            "Chord Spacing Test":"ChordSpacingTest.musicxml",
```

Comment:
maybe move this one down to the test files so we have the Beethoven one as default

Reply:
Yeah, I wasn't sure where to put that.  I had it at the top so it would load first for me in the demo and just left it there.  I'm thinking I'll just move it down just below the first chord test.